opendemic/scheduler/confirmed_cases.py

```diff
@@ -0,0 +1,115 @@
+from config.config import CONFIG, logger
+import os
+import pandas as pd
+import uuid
+import numpy as np
+import tarfile
+from tarfile import TarFile, ExFileObject
+import requests
+import sqlalchemy
+from sqlalchemy.engine.base import Engine
+from io import BytesIO
+from typing import List
+
+_CONFIRMED_CASE_DATA_URL: str = "https://github.com/beoutbreakprepared/nCoV2019/blob/master/latest_data/latestdata.tar.gz?raw=true"
+
+
+def generate_db_uri():
+	return 'mysql://' + CONFIG.get('rds-aurora-mysql-opendemic-username') + ':' + \
+		   CONFIG.get('rds-aurora-mysql-opendemic-password') + '@' + \
+		   CONFIG.get('rds-aurora-mysql-opendemic-host') + ':' + \
+		   CONFIG.get('rds-aurora-mysql-opendemic-port') + '/' + \
+		   CONFIG.get('rds-aurora-mysql-opendemic-database')
+
+
+def get_existing_source_ids() -> List[str]:
+	logger.debug("Getting existing source ids")
+	database_connection: Engine = sqlalchemy.create_engine(generate_db_uri())
+	with database_connection.begin() as cnx:
+		sql_query = sqlalchemy.text("""
+			SELECT `source_id`
+			FROM `humans`
+			WHERE `source` = 'beoutbreakprepared'
+		""")
+		result = cnx.execute(sql_query)
+		result_as_list: list = result.fetchall()
+		source_ids_to_remove: str = [item[0] for item in result_as_list]
+	return source_ids_to_remove
+
+
+def clean_data(data: pd.DataFrame) -> pd.DataFrame:
+	logger.debug("Cleaning dataset...")
+	data: pd.DataFrame = data[~pd.isna(data['ID'])]
+	data: pd.DataFrame = data[~pd.isna(data['date_confirmation'])]
+	data['date_confirmation'] = pd.to_datetime(data['date_confirmation'].apply(lambda x: x[:10]))
+	data['created'] = data['date_confirmation']
+	data['modified'] = data['date_confirmation']
+	data['latitude'] = pd.to_numeric(data['latitude'])
+	data['longitude'] = pd.to_numeric(data['longitude'])
+	data['biological_sex'] = data['sex'].apply(lambda x: 'F' if x == 'female' else ('M' if x == 'male' else np.nan))
+	data['symptom'] = 'verified_covid19'
+	data['source'] = 'beoutbreakprepared'
+	data['source_id'] = data['ID']
+	data['id'] = data['source_id'].apply(lambda x : str(uuid.uuid4()))
+	data['human_id'] = data['id']
+	try:
+		source_ids_to_remove = get_existing_source_ids()
+	except Exception as e:
```

Comment:
Not sure I see the point of catching an exception, to then raise it again. I believe that if you don't catch if you get the same result.

Reply:
That was intended as a placeholder. Forgot to add the logger there. Good catch